swagger.json

```diff
@@ -622,14 +622,12 @@
     "/data/opensearch": {
       "get": {
         "summary": "OpenSearch endpoint to receive standardized data search results.",
-        "description": "This service offers more complex search functionality and returns results in an OpenSearch compliant RSS XML format.",
+        "description": "This service offers more complex search functionality and returns results in an OpenSearch compliant Atom XML format.",
```

Comment:
Atom seems more flexible, e.g. you can add information about source/rights, and is basically the successor of RSS. In case we want to allow XML, Atom AND RSS, we should remove the specific information here.